docs/csharp/programming-guide/concepts/serialization/index.md

```diff
@@ -12,57 +12,69 @@ This illustration shows the overall process of serialization:
 
-### Basic serialization
+If a serialized class contains references to objects of other classes that are marked <xref:System.SerializableAttribute>, those objects will also be serialized.
 
-The only requirement in basic serialization is that the object has the <xref:System.SerializableAttribute> attribute applied. The <xref:System.NonSerializedAttribute> can be used to keep specific fields from being serialized.
+### Basic and custom serialization
 
-When you use basic serialization, the versioning of objects may create problems. You would use custom serialization when versioning issues are important. Basic serialization is the easiest way to perform serialization, but it does not provide much control over the process.
+Binary and XML serialization can be performed in two ways, basic and custom.
 
-### Custom serialization
+Basic serialization uses the .NET Framework to automatically serialize the object. The only requirement is that the class has the <xref:System.SerializableAttribute> attribute applied. The <xref:System.NonSerializedAttribute> can be used to keep specific fields from being serialized.
 
-In custom serialization, you can specify exactly which objects will be serialized and how it will be done. The class must be marked <xref:System.SerializableAttribute> and implement the <xref:System.Runtime.Serialization.ISerializable> interface.
+When you use basic serialization, the versioning of objects may create problems. You would use custom serialization when versioning issues are important. Basic serialization is the easiest way to perform serialization, but it does not provide much control over the process.
 
-If you want your object to be deserialized in a custom manner as well, you must use a custom constructor.
+In custom serialization, you can specify exactly which objects will be serialized and how it will be done. The class must be marked <xref:System.SerializableAttribute> and implement the <xref:System.Runtime.Serialization.ISerializable> interface. If you want your object to be deserialized in a custom manner as well, use a custom constructor.
 
 ## Designer serialization
 
 Designer serialization is a special form of serialization that involves the kind of object persistence associated with development tools. Designer serialization is the process of converting an object graph into a source file that can later be used to recover the object graph. A source file can contain code, markup, or even SQL table information.
 
 ## <a name="BKMK_RelatedTopics"></a> Related Topics and Examples  
+
+[System.Text.Json overview](../../../../standard/serialization/system-text-json-overview.md)
```

Comment:
nit: Consider adding a one-sentence description for each of these additional resources like the existing ones below.